<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>capture_main</class>
 <widget class="QWidget" name="capture_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scene Capture</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget[bgoverride=&quot;true&quot;] {
	background-color: #333;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin" stdset="0">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="setup_tab">
      <attribute name="title">
       <string>Setup</string>
      </attribute>
      <layout class="QVBoxLayout" name="setupTab_verticalLayout">
       <property name="margin" stdset="0">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="help_groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Help</string>
         </property>
         <layout class="QVBoxLayout" name="help_verticalLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="margin" stdset="0">
           <number>6</number>
          </property>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="plainText">
             <string>This tool captures snapshots which store the state of certain nodes in the scene.

On the first run, a set named 'sceneCapture_set1' will be created containing the current selection. Choose what to capture by adding or removing objects from this set. All keyable attributes of any transform or shape nodes will be captured.

A thumbnail image preview of the capture will be stored in the 'playblasts/sceneCapture/&lt;Scene&gt;' subfolder of the current Maya project. If there is a camera in the set, this will be used for snaphots. If there is more than one camera, it can be selected from the options below. If the set contains no cameras, the snapshot cam will default to 'persp'. The image dimensions set in the Render Globals will be used.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="options_groupBox">
         <property name="title">
          <string>Options</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="set_label">
            <property name="text">
             <string>Capture set:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="set_comboBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="snapCam_label">
            <property name="text">
             <string>Snapshot camera:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="snapCam_comboBox"/>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="noWarnings_checkBox">
            <property name="text">
             <string>Suppress warnings</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="setEditor_pushButton">
            <property name="toolTip">
             <string>Copy transform from last selected object</string>
            </property>
            <property name="text">
             <string>Edit set members...</string>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="tools_groupBox">
         <property name="title">
          <string>Tools</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="0" column="1">
           <widget class="QPushButton" name="xformPaste_pushButton">
            <property name="toolTip">
             <string>Paste transform to selected object(s)</string>
            </property>
            <property name="text">
             <string>Paste transform</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/paste.png</normaloff>:/icons/paste.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="xformSwap_pushButton">
            <property name="toolTip">
             <string>Swap positions of two objects</string>
            </property>
            <property name="text">
             <string>Swap transforms</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/swap.png</normaloff>:/icons/swap.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="xformCopy_pushButton">
            <property name="toolTip">
             <string>Copy transform from last selected object</string>
            </property>
            <property name="text">
             <string>Copy transform</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/copy.png</normaloff>:/icons/copy.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="data_groupBox">
         <property name="title">
          <string>Data management</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="import_pushButton">
            <property name="toolTip">
             <string>Import capture data from JSON file</string>
            </property>
            <property name="text">
             <string>Import JSON...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/import.png</normaloff>:/icons/import.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="export_pushButton">
            <property name="toolTip">
             <string>Export capture data to JSON file</string>
            </property>
            <property name="text">
             <string>Export JSON...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/export.png</normaloff>:/icons/export.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="delData_pushButton">
            <property name="toolTip">
             <string>Delete all external capture data for this scene</string>
            </property>
            <property name="text">
             <string>Delete saved data</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="captures_tab">
      <attribute name="title">
       <string>Captures</string>
      </attribute>
      <layout class="QVBoxLayout" name="capturesTab_verticalLayout">
       <property name="margin" stdset="0">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="newCapture_GroupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Create new capture</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="captureName_label">
              <property name="text">
               <string>Name:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="captureName_lineEdit">
              <property name="text">
               <string/>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="dragEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="capture_toolButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Capture</string>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Capture</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/icons/capture.png</normaloff>:/icons/capture.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="btn_renderAll">
              <property name="text">
               <string>On</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_renderNone">
              <property name="text">
               <string>Off</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_renderInvert">
              <property name="text">
               <string>Invert</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="captures_groupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Captures</string>
         </property>
         <property name="bgoverride" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="capturesGroup_verticalLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="margin" stdset="0">
           <number>6</number>
          </property>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <property name="bgoverride" stdset="0">
             <bool>false</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>813</width>
               <height>16</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="bgoverride" stdset="0">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="captures_verticalLayout">
              <property name="spacing">
               <number>4</number>
              </property>
              <property name="margin" stdset="0">
               <number>0</number>
              </property>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="render_groupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Create preview render of each checked set</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="verticalSpacing">
           <number>2</number>
          </property>
          <property name="margin" stdset="0">
           <number>6</number>
          </property>
          <item row="3" column="0">
           <widget class="QLabel" name="label_35">
            <property name="text">
             <string>Prefix</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="margin" stdset="0">
              <number>0</number>
             </property>
             <item>
              <widget class="QSpinBox" name="width_spinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>99999</number>
               </property>
               <property name="value">
                <number>2048</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="height_spinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>99999</number>
               </property>
               <property name="value">
                <number>1536</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QPushButton" name="render_button">
            <property name="palette">
             <palette>
              <active>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>170</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>170</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>170</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>RENDER</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QCheckBox" name="render_overrideSize">
            <property name="text">
             <string>Override Size</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_33">
            <property name="text">
             <string>W/H</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QComboBox" name="filetype_comboBox">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>jpg</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>exr</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="path_lineEdit">
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="render_tab">
      <attribute name="title">
       <string>Render</string>
      </attribute>
      <layout class="QVBoxLayout" name="renderTab_verticalLayout">
       <property name="margin" stdset="0">
        <number>6</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
